src/app/ui/question/question.component.ts

```diff
@@ -46,8 +48,8 @@ export class QuestionComponent implements OnInit {
   }
 
   markAsSolved() {
-    this.db.addSolvedQuestion(this.userId, {linkedQuestionId: this.qId}).then(() => {
-      this.snackBar.open(`This question was added to your solved question list!`, 'close', {duration: 3000});
+    this.db.addSolvedQuestion(this.userId, {linkedQuestionId: this.qId, difficulty: -1}).then(() => {
+      document.getElementById('openModal').click();
```

Comment:
We are not working with native JS here..

Reply:
I don't know how to solve it. How critical is it?

Reply:
I don't expect you to know. Read in the internet, see example in solution component and convert it to ng-bootstrap - it will be cleaner and easier.